redfish-core/lib/hypervisor_system.hpp

```diff
@@ -401,7 +420,10 @@ inline void parseInterfaceData(
     jsonResponse["@odata.id"] =
         "/redfish/v1/Systems/hypervisor/EthernetInterfaces/" + ifaceId;
     jsonResponse["InterfaceEnabled"] = true;
-    jsonResponse["MACAddress"] = ethData.mac_address;
+    if (!ethData.mac_address.empty())
```

Comment:
Why the empty check for Mac Address?

Reply:
This is because, the redfish validator failed if the MAC address is null..
ERROR - MACAddress: String '' does not match pattern ''^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$''

Reply:
We had some discussion here: 
https://ibm-systems-power.slack.com/archives/G3WAVQYTH/p1636031554243000?thread_ts=1635866209.193500&cid=G3WAVQYTH

I am still struggling here. 
Can you explain why MacAddress would be empty ?

Reply:
IMO, mac addr field can be removed from the redfish resp.
Because, do we actually need this in the redfish resp for these vmi interfaces?
It is optional in the schema as well.

This is included in the Ethernet interfaces redfish endpoint.

Reply:
MAC address is removed from the redfish response.